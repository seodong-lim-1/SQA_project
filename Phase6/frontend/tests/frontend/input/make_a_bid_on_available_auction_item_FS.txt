login
testFSUser
password
bid
itemToBidOn
sellerUserTest
20
logout
