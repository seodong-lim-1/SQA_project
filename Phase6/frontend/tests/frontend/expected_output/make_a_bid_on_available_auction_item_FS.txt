Welcome to the Online Auction Sales Service!

Please enter login to Login.
login - Login

***LOGIN***
Enter Username: 
Enter Password: 
Welcome Full-Standard User: testFSUser!

***MENU***
logout - Logout
advertise - Put up item for Auction
bid - Make a bid on an available item
addcredit - Add Credit
accounts - Display all active accounts
listitems - List Items Up For Auction
***MAKING A BID***

Enter Item Name: 
Enter seller's username: 
Currrent Highest Bid: 10.000000
Enter New Bid: A bid has been successfully made for this item!

NOTE: You've reached your session transaction limit. Please enter logout to log out and log back in to continue.

***MENU***
logout - Logout
advertise - Put up item for Auction
bid - Make a bid on an available item
addcredit - Add Credit
accounts - Display all active accounts
listitems - List Items Up For Auction

You have successfully logged out.
